.azure/pipelines/helix-test.yml

```diff
@@ -0,0 +1,27 @@
+# Don't run CI for this config yet. We're not ready to move official builds on to Azure Pipelines
+trigger: none
+
+# Run PR validation on all branches
+pr:
+  branches:
+    include:
+    - '*'
+
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Helix_flaky_x64
+    jobDisplayName: 'Tests: Flaky Helix x64'
+    agentOs: Windows
+    timeoutInMinutes: 240
+    steps:
+    - script: .\restore.cmd -ci
+      displayName: Restore
+    - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
```

Comment:
- what does `$(IsRequiredCheck)` mean here?
- shouldn't this command include `/p:IsFlakyRun=true` (perhaps instead)? (global prop will override setting in Helix.props)

Reply:
Doh good catch, when I synced the build command I stomped on the IsFlakyRun rip, rerunning

Reply:
We do want is required check because we want this to just be running flaky tests against the 2 required queues, basically like the PR validation normally does